### PR TITLE
Revert "Medical Cyborg Gripper Upgrade"

### PR DESCRIPTION
**Pull request by thegoldencat413**
Reverts yogstation13/Yogstation#20113

I really don't think that cyborgs should be able to do chemistry and virology, this was one of the only weaknesses of the medborg in the past, which was good, because it encouraged borgs to be a little bit dependent on players in order to do certain things, while still allowing them to be extremely strong at certain things. 
Currently borgs can do surgery with 100% success no matter the conditions, and can synthesize infinite basic healing chems, and with upgrades, infinite rare healing chems. This already makes them better than a doctor in most situations, not to mention their all access making them better than a paramedic as well. The only roles they didn't eclipse were chemistry and virology, requiring some amount of interaction by the crew. This basically allows a single ghost role to be better in every way at every single medical role, on top of basic borg things like built in sensor monitors, AA, and weapons. 

The focus of the game should be on the players I think, and in my opinion, borgs should be supplemental to them, they shouldn't be better than them at absolutely everything.

a potential counter-argument here is "Cyborgs are subject to laws", and while that's true, under most law sets would be incentivized to make chemicals instead of the chemist or another actual crew member.

the only other counter argument is "they can be flashed", I guess, but that doesn't really have anything to do with doing chemistry

you could also say "it makes sense, because a cyborg is a perfect machine, which should be better than a human" but I think that's just really bad design, the players and crew members should be the ones who are doing things, because they are the crew that the game revolves around, also, I just hate "this is realistic so it should be this way".

if anything, cyborgs should be nudged down in power level due to their AA and ranged door interaction which just makes them blow other roles out of the water (janitor, paramedic), they really don't need to be able to take over ever facet of a department like this PR makes them able to

thanks for your time,

### CODE REVIEW

**Comment by azzzertyy** [comment]
i think theres better solutions than removing the gripper entirely

**Comment by thegoldencat413** [comment]
i think borgs being able to work IVs is good, maybe it can be worked into the organ bag's functionality?

i think the negative outweighs the positive on this particular PR though so in my opinion reverting it would be good

**Comment by wonderinghost** [comment]
I mean, rune is doing a lot of Borg changes because of grippers. It's been years since engiborgs got rped upgrades. Along with grippers, he also had asked what if we made rcds more dependent on resources that linked to the silo, so engi borgs don't shit out infinite walls.

If you two talked you guys can probably come up with something other than removing it

**Comment by wonderinghost** [comment]
Like say infinite chems

**Comment by MajManatee** [comment]
holy shit golden cat pr
In other news, 90% of chemists say stuff like "I've made all the chemicals we need" after making occuline, inasculate, and like a handful of burn/brute meds. Id rather have a borg able to go in and make actually useful stuff like plastic and virus cures, than a know it all crewmember that's actually useless.

**Comment by ToasterBiome** [comment]
@neeshacark @KingSuperBOB @Runian @azzzertyy @PibeCantina @warface1234455 please describe why this shouldn't be reverted

**Comment by LaOubliette** [comment]
i dont want borgs making chems, i already hate when CMOs waltz into chem and just take my job away from me without even asking. Giving someone else with AA to just walk in and do whatever they want is awful even when they aren't antag and considering they could be hacked at any time it seems actively unbearable for medbay.

**Comment by Runian** [comment]
TG managed to add a version of this back when harmful healing chemicals (like Aiuri and Libital) were just added as a roundstart item; the [PR](https://github.com/tgstation/tgstation/pull/45494) had little to no outrage and had no fears of cyborgs overtaking all of the chemistry or virology. If there are no chemists, then the cyborg just fills up the gap with no issue. If there is a chemist, they will simply tell them to leave and escalate with them if they don't. It is as simple as that.

Furthermore, this is an upgrade item and not a round start item. I went and checked the last 15 rounds if there was a chemist and if chemistry was interacted with at all. 3/15 of them had at least one chemist. 2/3 of the chemists went AFK or never came back to their lab later in the shift. 3/12 of the non-chemist rounds had a doctor break in to make something minor like plastic. If this is a common occurrence, I don't think this upgrade is gonna wipe out or suddenly ruin every chemistry mains once a medical cyborg gets a gripper late into the round.

But if you're so keen to prevent cyborgs from doing chemistry, you'll also need to revert #20045 as it enables service cyborgs to do chemistry as a round start feature as well.

**Comment by azzzertyy** [comment]
> @neeshacark @KingSuperBOB @Runian @azzzertyy @PibeCantina @warface1234455 please describe why this shouldn't be reverted

I just think theres better solutions to the problem, may require more work though.
I personally think mediborgs should be nerfed all around, give them slower tool speed, slower action speed, whathaveyounot rather than just an all access medical doctor that can do every job better than a human
reverting this to me personally seems like a cope to deal with the fact mediborgs are just better than doctors in every way the way they are right now, and is misplaced on them being able to grip things rather than their actual problems, which are the 100% successful tools, having access to chem machines and viro machines (they shouldnt), and probably their actual speed (theyre better at being paramedics)

File: code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -866,7 +866,6 @@
 	. = ..()
 	if(wrapped)
 		var/mutable_appearance/wrapped_appearance = mutable_appearance(wrapped.icon, wrapped.icon_state)
-		wrapped_appearance.overlays = wrapped.overlays.Copy()
```
Comment (Moltijoe) [inline code/game/objects/items/robot/robot_items.dm:869]: this probably shouldn't be reverted since it's unrelated to the functionality of the gripper

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]
Actually, that only matters if they're holding something with an overlay, which is literally only medical stuff

**Comment by neeshacark** [comment]
i just disagree fundamentally that grippers for medborg is bad, honestly.

**Comment by ToasterBiome** [comment]
> i just disagree fundamentally that grippers for medborg is bad, honestly.

This is not a reason, "I disagree" to asking why you disagree lol

**Comment by ToasterBiome** [comment]
Seeing this (or lack of seeing this honestly) makes it seem like it's actually not that large of an issue.

**Pull request closed without merging**